wbooks/src/App/constants/endpoints.js

```diff
@@ -1,5 +1,9 @@
 export default {
   login: 'users/sessions/',
   signup: 'users/',
-  books: '/books'
+  books: '/books',
+  rents: '/books/id/rents',
+  wishList: '/users/id/wishes',
+  userInfo: '/users/me',
+  comments: '/books/id/comments'
```

Comment:
Also this consts, I don't see why they should all be in the same file.
Define them where you use them.